scripts/turtles-quickstart.sh

```diff
@@ -0,0 +1,297 @@
+kubectl wait --for=condition=Ready node/$RANCHER_CLUSTER_NAME-control-plane
+kubectl get nodes -o wide
+
+show_step "Install Ingress NGINX controller on ${RANCHER_CLUSTER_NAME}"
+
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm repo update
+helm install --create-namespace --namespace ingress-nginx ingress-nginx ingress-nginx/ingress-nginx --set controller.hostNetwork=true
+
+show_step "Install Cert Manager on ${RANCHER_CLUSTER_NAME}"
+
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --create-namespace \
+    --set crds.enabled=true
+
+show_step "Install Rancher latest version on ${RANCHER_CLUSTER_NAME}"
+
+helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+helm repo update
+helm install rancher rancher-latest/rancher \
+    --namespace cattle-system \
+    --create-namespace \
+    --set bootstrapPassword="$RANCHER_PASSWORD" \
+    --set replicas=1 \
+    --set hostname="$RANCHER_HOSTNAME" \
+    --set ingress.ingressClassName=nginx \
+    --version="$RANCHER_VERSION" \
+    --wait
+
+kubectl rollout status deployment rancher -n cattle-system --timeout=180s
+
+cat > rancher-settings.yaml <<EOF
+apiVersion: management.cattle.io/v3
+kind: Setting
+metadata:
+  name: server-url
+value: https://${RANCHER_HOSTNAME}
+---
+apiVersion: management.cattle.io/v3
+kind: Setting
+metadata:
+  name: first-login
+value: "false"
+---
+apiVersion: management.cattle.io/v3
+kind: Setting
+metadata:
+  name: eula-agreed
+value: "2023-08-16T20:39:11.062Z"
+---
+apiVersion: management.cattle.io/v3
+kind: Setting
+metadata:
+  name: agent-tls-mode
+value: "system-store"
+EOF
+
+kubectl apply -f rancher-settings.yaml
+
+kubectl rollout status deployment rancher -n cattle-system --timeout=180s
+
+show_step "Waiting for Rancher to be READY ..."
+
+until [[ $(curl -ks https://${RANCHER_HOSTNAME}/ping) == "pong" ]]; do
+    sleep 1
+done
+echo "Received pong!"
+
+show_step "Install Rancher Turtles on ${RANCHER_CLUSTER_NAME}"
+
+helm repo add turtles https://rancher.github.io/turtles
+helm repo update
+helm install rancher-turtles turtles/rancher-turtles --version v0.16.0 \
+    -n rancher-turtles-system \
+    --set rancherTurtles.managerArguments={--insecure-skip-verify} \
+    --set turtlesUI.enabled=true \
+    --dependency-update \
+    --create-namespace --wait \
+    --timeout 180s
+
+show_step "Waiting for deployment capi-controller-manager to be created..."
+
+until kubectl get deployment capi-controller-manager -n capi-system > /dev/null 2>&1; do
+  sleep 1
+done
+kubectl rollout status deployment capi-controller-manager -n capi-system --timeout=180s
+
+# Enable the experimental Cluster topology feature.
+export CLUSTER_TOPOLOGY=true
+
+show_step "Install CAPI Docker provider on ${RANCHER_CLUSTER_NAME}"
+
+clusterctl init --infrastructure docker
```

Comment:
we don't support `clusterctl`, all providers have to be installed with `CAPIProvider`

Reply:
it was done